### PR TITLE
DEVAS 1843: form to write fiction is bigger. Can now save fiction wit…

### PR DESCRIPTION
**Pull request by cedrtang**
- form to write fiction is bigger
- Can now save fiction with either title or content

### CODE REVIEW

File: assembl/static2/js/app/components/debate/common/topPostForm.jsx [hunk 1]
```diff
@@ -137,6 +140,9 @@ class TopPostForm extends React.Component<TopPostFormProps, TopPostFormState> {
             this.setState({ submitting: false });
           });
       });
+    } else if (publicationState === PublicationStates.DRAFT && (!subject && bodyIsEmpty)) {
+      displayAlert('warning', I18n.t('debate.brightMirror.fillEitherTitleContent'));
+      this.setState({ submitting: false });
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/components/debate/common/topPostForm.jsx]: Functional form, perhaps. https://medium.freecodecamp.org/functional-setstate-is-the-future-of-react-374f30401b6b

File: assembl/static2/js/app/components/debate/brightMirror/instructionView.jsx [hunk 2]
```diff
@@ -52,6 +52,7 @@ const InstructionView = ({
       bodyMaxLength={NO_BODY_LENGTH}
       draftable
       draftSuccessMsgId="debate.brightMirror.draftSuccessMsg"
+      fullscreen
```
Comment (ayyazdaniaryan) [inline assembl/static2/js/app/components/debate/brightMirror/instructionView.jsx]: How would fullscreen be set upstream?

Comment (cedrtang) [reply assembl/static2/js/app/components/debate/brightMirror/instructionView.jsx]: @ayyazdaniaryan Doesn't need to go higher, it s for the TopFormContainer that is already a higher up component

File: assembl/static2/js/app/components/debate/common/topPostForm.jsx (hunk 1 above)
Comment (MilanBarande) [reply assembl/static2/js/app/components/debate/common/topPostForm.jsx]: Isn't it only useful when you use the previousState (eg to toggle a boolean or increase a counter) ?

**Pull request merged**